assets/css/_diversion_page.scss

```
@@ -97,6 +97,11 @@
       display: none;
     }
   }
+
+  ol,
+  ul {
+    margin-top: unset;
+  }
```

note(non-blocking: I think this is fine for now): This is my biggest uncertainty, mainly because we expect to have the reboot handle a lot of this for us.

question: Could you explain why to use `unset` instead of `0`?
```suggestion
  ol,
  ul {
    margin-top: 0;
  }
```

---

🫤 Emmm I have no justification for this one! I started with unset to get a sense of what stacked margin modifiers were getting applied to the element, and suppose I didn't revisit it? `0` is correct for my uses here. I agree that reboot will change all of this, which is bothersome